Makefile

```diff
@@ -19,9 +19,14 @@ clean:
 # TODO: add string manipulation => add code snippets in line_protocol.proto, main.pp: action_handler/ registration_handler
 add_driver: 
 	# add <new_driver>.cpp to src/drivers
-	sed 's/template_driver/$(driver_name)/' ./template_files/template_driver.cpp > ./src/drivers/$(driver_name).cpp
+	sed 's/template_driver/$(driver_name)/' ./tools/template_files/template_driver.cpp > ./src/drivers/$(driver_name).cpp
 	# <new_driver>.h to include/drivers
-	sed 's/template_driver/$(driver_name)/' ./template_files/template_driver.h > ./include/drivers/$(driver_name).h
+	sed 's/template_driver/$(driver_name)/' ./tools/template_files/template_driver.h > ./include/drivers/$(driver_name).h
```

Comment:
Used `/tools` dir since it's best suited IMO.
From the [link](https://api.csswg.org/bikeshed/?force=1&url=https://raw.githubusercontent.com/vector-of-bool/pitchfork/develop/data/spec.bs#tld.tools) you posted in the [weekly log](https://github.com/Interactions-HSG/hypermedia-factory/wiki/%5BJonas%5D-Meeting-Logs#topics-to-discuss)
> The tools/ directory is designated for holding extra scripts and tools related to developing and contributing to the project. For example, turn-key build scripts, linting scripts, code-generation scripts, test scripts, or other tools that may be useful to a project develop.